app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -933,7 +964,8 @@ def prov_set_form_vars(req = nil)
       end
 
       if @edit[:wf].kind_of?(ManageIQ::Providers::Foreman::ConfigurationManager::ProvisionWorkflow) ||
-         @edit[:wf].kind_of?(PhysicalServerProvisionWorkflow)
+         @edit[:wf].kind_of?(PhysicalServerProvisionWorkflow) ||
```

Comment:
Note for a follow-up, these are both `MiqConfiguredSystemProvisionWorkflow` types